phi/assistant/assistant.py

```diff
@@ -94,10 +107,12 @@ class Assistant(BaseModel):
     # If use_tools = True, set read_chat_history and search_knowledge = True
     use_tools: bool = False
 
-    # -*- Important: this setting determines if the input messages are formatted
-    # If True, phidata will add the system prompt, references, and chat history
-    # If False, the input messages are sent to the LLM as is
-    format_messages: bool = True
+    #
+    # -*- Assistant Messages
+    #
+    # -*- List of messages added to the messages list after the system prompt.
+    # Use these for few-shot learning or to provide additional context to the LLM.
+    add_messages: Optional[List[Union[Dict, Message]]] = None
```

Comment:
This feels a bit confusing for me. Could be `additional_messages` instead? `add_messages` feel like a method for adding messages to a list.

Reply:
@sandangel agree, will update it to be `additional_messages`

Reply:
@sandangel looking for a better name for this param :/